forms/serializers/form.py

```diff
@@ -535,46 +535,39 @@ class Meta:
 
     @staticmethod
     def get_applicant(obj):
+        applicant_list = list()
         try:
             applicant_section = obj.entry_sections.get(
                 entries__field__identifier="hakija",
                 entries__field__section__identifier="hakijan-tiedot",
             )
-            applicant_type = applicant_section.sections.get(
+            applicant_type = applicant_section.entries.get(
                 field__identifier="hakija", field__section__identifier="hakijan-tiedot"
             ).value
             if applicant_type == "Yritys":
```

Comment:
As I commented previously, forms copied from the current default template use the values "1" (company) and "2" (person) for the two options, so neither will ever match in a real-world scenario. Either these need to be changed or the default template needs to be altered.

Reply:
No luck even with the newer version. I do actually think these are saved as strings, not as numbers, which may have  an effect.